### ISSUE TITLE
Grbl Controller 3.6.1 Settings-Apply error

### ISSUE THREAD

**Issue by garyrkey** [zapmaker/GrblHoming#105]
When a Value is changed in the Grbl Settings Pop-Up and Apply clicked, the value is applied to another Item. For example, when trying to change the setting for Z max rate ($112) the new value is applied to homing feed ($24). This can be observed on the (log?) panel under the Command text box on the main page. Instead of logging a $112="somevalue", the operation $24="somevalue".

**Comment by jonlidgard** [comment]
Yep, I can confirm. You have to enter the value on the command line.
